smaily_for_drupal/smaily_for_drupal.module

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @file
+ * Module file for smaily_for_durpal.
+ */
+
+use Drupal\Core\Routing\RouteMatchInterface;
+
+/**
+ * Implements hook_help().
+ */
+function smaily_for_drupal_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    case 'help.page.smaily_for_drupal':
+      $output = '';
+      $output .= '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('The Smaily for Drupal module allows you to integrate the Smaily API for your site. A newsletter subscription block can be configured and placed where possible.<br> More information about Smaily is available at the Smaily <a href=http://help.smaily.com/en/support/home> Knowledgebase.') . '</a></a></p>';
```

Comment:
Try to avoid having links inside translation phrases.

This can be written as:
```php
t('... More information about Smaily is available in the Smaily <a href=":knowledgebase_url">Knowledgebase</a>', array(':knowledgebase_url' => 'http://help.smaily.com/en/support/home'));
```

PS. You have 2 link closing tags outside of translation phrase. One should be removed and the other should be in the phrase.